src/utils/OpenPepXL.cpp

```diff
@@ -423,14 +482,15 @@ class TOPPOpenPepXL :
     Int max_precursor_charge = getIntOption_("precursor:max_charge");
     double precursor_mass_tolerance = getDoubleOption_("precursor:mass_tolerance");
     bool precursor_mass_tolerance_unit_ppm = (getStringOption_("precursor:mass_tolerance_unit") == "ppm");
+    IntList precursor_correction_steps = getIntList_("precursor:corrections");
```

Comment:
your main method is almost 1000 LOC, this is clearly too long. please consider extracting some functions